bindings/cpu_profiler.cc

```diff
@@ -132,45 +208,58 @@ class Profiler {
   static void DeleteInstance(void* data);
 };
 
-void Profiler::DeleteInstance(void* data) {
-  Profiler* profiler = static_cast<Profiler*>(data);
-  profiler->cpu_profiler->Dispose();
-  delete profiler;
-}
-
 class SentryProfile {
 private:
-  uint64_t started_at = 0;
+  uint64_t started_at;
   uint16_t heap_measurement_index = 0;
-  std::vector<uint64_t> heap_stats_ts;
-  std::vector<uint64_t> heap_stats_usage;
-  std::function<void(uv_timer_t*, uint64_t, v8::HeapStatistics&)> memory_sampler_cb;
-  const char* id;
-  ProfileStatus status;
+  uint16_t cpu_measurement_index = 0;
+
+  std::vector<uint64_t> heap_stats_ts = std::vector<uint64_t>(300);
+  std::vector<uint64_t> heap_stats_usage = std::vector<uint64_t>(300);
+
+  std::vector<uint64_t> cpu_stats_ts = std::vector<uint64_t>(300);
+  std::vector<double> cpu_stats_usage = std::vector<double>(300);
```

Comment:
you can simplify all of these to remove the assignments

e.g. `std::vector<uint64_t> heap_stats_ts(300)`

Reply:
Egh, I tried and it complains with `expected a type specifier`. I'll remove the assignment and add it to the initializer list